payjoin-cli/src/app.rs

```diff
@@ -91,7 +91,8 @@ impl App {
         let psbt = Psbt::from_str(&psbt).with_context(|| "Failed to load PSBT from base64")?;
         log::debug!("Original psbt: {:#?}", psbt);
         let fallback_tx = psbt.clone().extract_tx();
-        let (req, ctx) = payjoin::send::RequestBuilder::from_psbt_and_uri(psbt, uri)
+        let version = 1;
+        let (req, ctx) = payjoin::send::RequestBuilder::from_psbt_and_uri(psbt, uri, version)
```

Comment:
motivation > we need a way to provide the sender to specify the version that will be appended in the url